scrapers/AmateureExtreme.yml

```diff
@@ -0,0 +1,32 @@
+name: "AmateureXtreme"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - amateure-xtreme.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1[@class='info-section--title']
+      URL: //link[@rel="canonical"]/@href
+      Date:
+        selector: //div[@class='posted-info meta-description small']/div[@class='float-left']/span[@class='text-nowrap']/span[@class='fa5-text']
+        postProcess:
+          - replace:
+              - regex: $
+                with: $1-01-01
+      Tags:
+        Name: //div[@class='tags mt-3']/a | //div[@class='posted-info meta-description small']/div[@class='float-left']/span[2]/span[@class='fa5-text']/a

```

Comment:
simpler ?
```yaml
        Name: //a[contains(@href,'collections?')]
```

Reply:
Tested, works. I need to get more familiar with contains...